Add docs on how to use the fissile build release-images (#524)
cloudfoundry-incubator/fissile: docs/build-docker-imgs.md

# Building docker images

The `fissile build` command comes with a helpful subcommand for building docker images from a **BOSH** release.

The command is `fissile build release-images`. See the [docs](/generated/fissile_build_release-images.md)

## How to use it?

The following are the flags that you need to provide, in order to generate assets and building the Docker image.

- `--stemcell`

    You need to provide a reference to a docker image, containing the stemcell you want to use, as the underlying layer. For example:

  ```sh
  --stemcell=splatform/fissile-stemcell-opensuse:42.3-36.g03b4653-30.80
  ```

>   **Note**: You docker stemcell image requires to have two specific labels. This are `stemcell-flavor` and `stemcell-version`.

Reviewer: Tiny grammar nit:
```suggestion
  **Note**: You docker stemcell image is required to have two specific labels. They are `stemcell-flavor` and `stemcell-version`.
```

- `--name`

>     This is the name of the BOSH release. The same value you assign, when defining this release on a BOSH deployment manifest.

Reviewer: ```suggestion
    This is the name of the BOSH release. This should be the value defined in the BOSH [deployment manifest](https://bosh.io/docs/manifest-v2/#releases).
```
Please make similar wording changes for the next three values.
